src/js/utils/elementUtils.js

```diff
@@ -0,0 +1,22 @@
+import { Document as document } from "./server-safe-globals";
+
+export const getElementBySelector = (selector, nameStr, root = document) => {
+    const matches = root.querySelectorAll(selector);
+
+    if (!matches.length) return undefined;
+
+    if (matches.length > 1) {
+        const namePhraseStr = nameStr ? ` ${nameStr} ` : '';
+        console.info(`Multiple elements matches found for${namePhraseStr}selector ${selector}. Will yield first match.`);
```

Comment:
If this warning is really the only reason the attribute name is required for this function then I think we should simplify. I would be fine with killing the warning, but otherwise you could just prefix with `Media Chrome: ` to give some context even if it's not perfect.

It does feel like we've remade `document.querySelector` for the sake of adding a warning. I think I'd rather be clear in docs than over complicate this function for that purpose.

Reply:
ok. Thought this is what we landed on in our previous convo, but will switch back to `querySelector` and simplify.